src/main/features/requirements/controller/ca-teamScale.ts

```diff
@@ -32,13 +33,37 @@ export const CA_GET_TEAM_SCALE = async (req: express.Request, res: express.Respo
   try {
     // get the stored value from session. If not found, call api
     const assessmentDetail = await GET_ASSESSMENT_DETAIL(SESSION_ID, assessmentId);
-    if (assessmentDetail.dimensionRequirements.length > 0) {
-      const optionId = assessmentDetail.dimensionRequirements[0].requirements[0]['requirement-id'];
-      const objIndex = caTeamScale.form.radioOptions.items.findIndex(obj => obj.value === optionId);
+    const assessmentURL=`assessments/${assessmentId}`;
+    const assessmentData = await TenderApi.Instance(SESSION_ID).get(assessmentURL);
+    const externalID=assessmentData.data['external-tool-id']
+    const Data=assessmentData.data.dimensionRequirements.filter(x=>x["dimension-id"]===4);
```

Comment:
Please use code like below 
let Data;
if(assessmentData !=null && assessmentData.data != null && assessmentData.data.dimensionRequirements !=null )
{
Data = assessmentData.data.dimensionRequirements.filter(x=>x["dimension-id"]===4);
}
Sometime will get exception because of null value